### PR TITLE
[DBIO-1699 & DBIO-1700]genetic analysis verification

### PR DESCRIPTION
**Pull request by JackyRahman**
### JIRA Link
[https://blocksphere2020.atlassian.net/browse/DBIO-1699](https://blocksphere2020.atlassian.net/browse/DBIO-1699)
[https://blocksphere2020.atlassian.net/browse/DBIO-1700](https://blocksphere2020.atlassian.net/browse/DBIO-1700)

### Changelog / Description

- [create model and command genetic Analysts](https://github.com/debionetwork/debio-backend/commit/92a2f21a1dbcb9f4d8dc5ea2fa5df4b25d4eff3a)
- [feat: add verification genetic analyst to verification service](https://github.com/debionetwork/debio-backend/commit/cb7542a27a9dac9a542327c00447d21195aa2461)
- [feat: add verification genetic analyst to verification service](https://github.com/debionetwork/debio-backend/commit/cb7542a27a9dac9a542327c00447d21195aa2461)
- [tests: update verification service unit testing for genetic analysts](https://github.com/debionetwork/debio-backend/commit/c6e88dc513141c9279550f6296bb26c73a56a00f)
- [tests: update verification controller unit testing for genetic analysts](https://github.com/debionetwork/debio-backend/commit/27d1c023a7d45a996e36d3b9ee7f3e863d1f12f1)

### CODE REVIEW

File: src/endpoints/substrate-endpoint/services/genetic-analysis.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          query: {
+            match: {
+              genetic_analyst_tracking_id: {
+                query: genetic_analyst_tracking_id,
+              },
+            },
+          },
+        },
+      });
+      hitsGeneticAnalys = geneticAnalysis.body.hits.hits;
```
Comment (agustinustheo) [inline src/endpoints/substrate-endpoint/services/genetic-analysis.ts:25]: Please fix the naming typo here.

File: src/endpoints/verification/verification.service.ts [hunk 2]
```diff
@@ -49,4 +51,15 @@ export class VerificationService {
     };
     return await this.rewardService.insert(dataInput);
   }
+
+  async verificationGeneticAnalyst(accountId: string, verificationStatus: string) {
+    await updateGeneticAnalystVerificationStatus(
+      this.subtrateService.api,
+      this.subtrateService.pair,
+      accountId,
+      <GeneticAnalystsVerificationStatus>verificationStatus,
```
Comment (agustinustheo) [inline src/endpoints/verification/verification.service.ts:60]: Why is there a `<` here?

File: test/unit/endpoints/verification/verification.service.spec.ts [hunk 3]
```diff
@@ -71,6 +74,25 @@ describe('Verification Service Unit Tests', () => {
     expect(verificationService).toBeDefined();
   });
 
+  it('shoul update genetic analyst status', async () => {
+    const NOW = 0;
+    const ACCOUNT_ID = 'ACCOUNT_ID';
+    const VERIFICATION_STATUS = 'Verified';
+
+    const RESULTS = await verificationService.verificationGeneticAnalyst(
```
Comment (agustinustheo) [inline test/unit/endpoints/verification/verification.service.spec.ts:82]: Remove unused `RESULTS`, you don't need to store the results of `verificationGeneticAnalyst` into the variable.

File: test/unit/endpoints/verification/verification.service.spec.ts [hunk 4]
```diff
@@ -71,6 +74,25 @@ describe('Verification Service Unit Tests', () => {
     expect(verificationService).toBeDefined();
   });
 
+  it('shoul update genetic analyst status', async () => {
+    const NOW = 0;
```
Comment (agustinustheo) [inline test/unit/endpoints/verification/verification.service.spec.ts:78]: Please remove unused `NOW` variable.

**Review by agustinustheo — CHANGES_REQUESTED** [review]
Minor bug fixes, otherwise, done.

File: src/endpoints/verification/verification.service.ts (hunk 2 above)
Comment (JackyRahman) [reply src/endpoints/verification/verification.service.ts:60]: for verification_status same with GeneticAnalystsVerificationStatus model...

File: src/endpoints/substrate-endpoint/services/genetic-analysis.ts (hunk 1 above)
Comment (agustinustheo) [reply src/endpoints/substrate-endpoint/services/genetic-analysis.ts]: Still typo

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**